code/controllers/subsystems/jobs.dm

```diff
@@ -476,8 +476,8 @@ SUBSYSTEM_DEF(jobs)
 			G.prescription = 7
 	if(job.possible_backstories.len && H.client.prefs.backstory)
 		pick_backstory(job.possible_backstories, H)
+		H.set_squad_huds()
```

Comment:
?????

Reply:
Drunkeness  this is obviously not done and I have to relook at it again so ugh.